www/md_base/guanlecoja/config.coffee

```diff
@@ -88,4 +88,47 @@ gulp.task 'icons', ->
         ))
         .pipe(gulp.dest(path.join(config.dir.build, 'icons')))
 
+gulp.task 'processindex', ['index'], ->
+    indexpath = path.join(config.dir.build, 'index.html')
+    gulp.src ""
+        .pipe shell("buildbot processwwwindex -i '#{indexpath}'")
+
+gulp.task 'proxy', ['processindex'], ->
+    # this is a task for developing, it proxy api request to http://nine.buildbot.net
+    argv = require('minimist')(process.argv)
+    argv.host?= 'nine.buildbot.net'
+    argv.port?= 8020
+
+    fs = require 'fs'
+    path = require 'path'
+    http = require 'http'
+    httpProxy = require 'http-proxy'
+    proxy = httpProxy.createProxyServer({})
+    proxy.on 'proxyReq', (proxyReq, req, res, options) ->
+        delete proxyReq.removeHeader('Origin')
+        delete proxyReq.removeHeader('Referer')
+    proxy.on 'proxyRes', (proxyRes, req, res) ->
+        proxyRes.headers['Access-Control-Allow-Origin'] = '*'
+        console.log "[Proxy] #{req.method} #{req.url}"
+
+    server = http.createServer (req, res) ->
+        if req.url.match /^\/(api|sse)/
+            proxy.web req, res, {target: 'http://' + argv.host}
+        else if req.url.match /^\/ws/
+            proxy.ws req, res, {target: 'ws://' + argv.host}
+        else
```

Comment:
Great! But you dont fully manage the plugins. Each plugin have it's own directory of static files with one script and one CSS file. You need to tell the proxy where the file are in order to serve them.